io/net/ethtool_test.py.data/ethtool_test.yaml

```diff
@@ -3,6 +3,10 @@ peer_ip:
 host_ip:
 netmask:
 hbond:
+tx_channel: 
+rx_channel: 
+other_channel:
+combined_channel:
```

Comment:
the above input params are specific to `-L` option, so why are they in common,  can these be moved inside `Set-channels:` variant ?